packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -413,9 +413,8 @@ export async function runNpmInstall(
 
   try {
     await runNpmInstallSema.acquire();
-    const { cliType, packageJsonPath, lockfileVersion } = await scanParentDirs(
-      destPath
-    );
+    const { cliType, packageJsonPath, packageJson, lockfileVersion } =
+      await scanParentDirs(destPath, true);
```

Comment:
I dislike how little the extra `true` tells me about what the function is now doing that it wasn't before.

Reply:
Agreed. This style of signature is not great.